windows_registry/manifest.json

```
@@ -0,0 +1,39 @@
+{
+    "manifest_version": "2.0.0",
+    "app_uuid": "cc166a5c-6742-4811-b3e1-93dbec0ac5b2",
+    "app_id": "windows-registry",
+    "display_on_public_website": false,
```

```suggestion
    "display_on_public_website": false,
```
Just a reminder that this needs to be set to `true` to publish on the docs site

---

thanks! changing to true